Emrald-UI/src/components/forms/VariableForm/VariableFormContext.tsx

```diff
@@ -0,0 +1,262 @@
+import { createContext, PropsWithChildren, useContext, useState } from 'react';
+import { useWindowContext } from '../../../contexts/WindowContext';
+import { v4 as uuidv4 } from 'uuid';
+import { useVariableContext } from '../../../contexts/VariableContext';
+
+import { Variable } from '../../../types/Variable';
+import {
+  AccrualVarTableType,
+  DocVarType,
+  VariableType,
+  VarScope,
+} from '../../../types/ItemTypes';
+import { SelectChangeEvent } from '@mui/material';
+
+export interface AccrualStateItem {
+  stateName: string;
+  type: AccrualVarTableType;
+  accrualMult: number;
+  multRate: string;
+  accrualTable: number[][];
+}
+
+interface VariableFormContextType {
+  accrualStatesData?: AccrualStateItem[];
+  setAccrualStatesData: React.Dispatch<
```

Comment:
This isn't super important either but typically I think we should try and keep all the function items at the bottom and all the string items and such at the top.

Reply:
Those have now been moved down